lib/ex_doc/language/elixir.ex

```diff
@@ -440,7 +440,7 @@ defmodule ExDoc.Language.Elixir do
 
   @ref_regex ~r/^`(.+)`$/
 
-  defp custom_link(attrs, config) do
+  def custom_link(attrs, config) do
```

Comment:
This is the function I mention in the description. Where to put it, and the code it depends on?

Reply:
You can put it in the ExDoc.Autolink module :)

Reply:
Got it, thanks. I'll move it, and the code it depends on, to that one.

Reply:
ce25ca50